packages/contracts/contracts/registry/ens/ENSSubdomainRegistrar.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import "@ensdomains/ens-contracts/contracts/registry/ENS.sol";
+import "@ensdomains/ens-contracts/contracts/resolvers/Resolver.sol";
+import "../../core/component/Component.sol";
+
+/// @title A registrar for ENS subdomains
+/// @author Aragon Association - 2022
+/// @notice This contract registers ENS subdomains under a parent domain specified in the initialization process
+///         and maintains ownership of the subdomain since only the resolver address is set.
+///         This contract must either be the domain node owner or an approved operator of the node owner.
+///         During the subdomain initialization, the resolver specified in the parent domain is used.
+
+contract ENSSubdomainRegistrar is Component {
```

Comment:
- Are we deploy this contract? 
if yes can you also create deploy script for it?
if not, how it is intended to be used?

Reply:
It will be deployed and needs the Admin DAO you are using for your registries as well.
Accordingly, it makes more sense to write the deploy script after rebasing your branch on top to prevent merge conflicts and use things you coded already instead of duplicating work and we shouldn't make this PR bigger than necessary.

Reply:
Can we create a follow-up task for that, then?

Reply:
ok, noted.

Reply:
I added this as a note to [APP-407](https://aragonassociation.atlassian.net/browse/APP-407) because it will be deployed together with the DAO registry.